src/peft/tuners/lora/gptq.py

```diff
@@ -106,10 +107,15 @@ def dispatch_gptq(
     else:
         target_base_layer = target
 
-    gptq_quantization_config = kwargs.get("gptq_quantization_config", None)
-    AutoGPTQQuantLinear = get_auto_gptq_quant_linear(gptq_quantization_config)
+    cfg = kwargs.get("gptq_quantization_config", None)
 
-    if AutoGPTQQuantLinear is not None and isinstance(target_base_layer, AutoGPTQQuantLinear):
+    if is_gptqmodel_available():
+        device_map = kwargs.get("device_map", None)
+        quant_linear = get_gptqmodel_quant_linear(cfg, device_map=device_map)
+    else:
+        quant_linear = get_auto_gptq_quant_linear(cfg)
```

Comment:
Would it make sense to suggest for users to move to gptqmodel here or is it too early for that?

Reply:
It's not too early but our plan is to deprecate autogptq but only want to do so for next release of PEFT, after gptqmodel is merged and no regressions of bugs are encountered in the wild since our deprecation plan covers transformers, optimum, and peft, I want to be extra careful about this.